From 5c198c35b02e2c0a2c75f862fca7051c78b318a7 Mon Sep 17 00:00:00 2001
From: Richard Fitzgerald <rf@opensource.wolfsonmicro.com>
Date: Thu, 23 Oct 2014 11:16:48 +0100
Subject: [PATCH 344/351] Documentation: wolfson-arizona: WM8998 inmode pdata
 settings

Describe the behaviour of the inmode pdata for the
muxed inputs on the WM8998.

Change-Id: I6069b4b7d38c04396ec2fc767c257f6f3b4c4321
Signed-off-by: Richard Fitzgerald <rf@opensource.wolfsonmicro.com>
---
 Documentation/sound/alsa/soc/wolfson-arizona.txt |   14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/Documentation/sound/alsa/soc/wolfson-arizona.txt b/Documentation/sound/alsa/soc/wolfson-arizona.txt
index da2f6d7..38b6d06 100644
--- a/Documentation/sound/alsa/soc/wolfson-arizona.txt
+++ b/Documentation/sound/alsa/soc/wolfson-arizona.txt
@@ -147,3 +147,17 @@ To change from a digital IN1A to an analogue IN1B follow this sequence:
 2) Change the IN1MUXL and IN1MUXR to IN1B
 3) Connect IN1 as an input to codec blocks
 
+The input mode pdata for WM8998 is defined as:
+
+ pdata      affects  legal values            behaviour
+ inmode[0]  IN1      ARIZONA_INMODE_DIFF (0) IN1A and IN1B analogue differential
+                     ARIZONA_INMODE_SE   (1) IN1A and IN1B analogue single-ended
+                     ARIZONA_INMODE_DMIC (2) IN1A digital, IN1B analogue differential
+                     ARIZONA_INMODE_DMIC |
+                       ARIZONA_INMODE_SE (3) IN1A digital, IN1B analogue single-ended
+
+ inmode[1]  IN2      ARIZONA_INMODE_DIFF (0) IN2A and IN2B analogue differential
+                     ARIZONA_INMODE_SE   (1) IN2A and IN2B analogue single-ended
+                     ARIZONA_INMODE_DMIC (2) IN2A digital, IN2B analogue differential
+                     ARIZONA_INMODE_DMIC |
+                       ARIZONA_INMODE_SE (3) IN2A digital, IN2B analogue single-ended
-- 
1.7.9.5

